### DISCUSSION TITLE
Live Preview

### DISCUSSION THREAD

**Discussion by samii-ullah** [GrapesJS/grapesjs#3820] [General]
Hello Everyone,
I am working on Grapejs application to build a website builder. Could you please tell me how to set up continuous deployment locally?
Like when I run my react app, it will render HTML pages and show the application running on other ports locally,
I will deploy it on cloud cone and map domain too but that will be 2nd milestone,
I want to see my application running on a browser, a website builder is generating code for the new website but I want my app to handle the rendering of HTML pages by ownself.
can anyone help me to make a road map?
Thank you in advance

**Comment by artf** [comment]
Probably you need to ask questions like this on Stack Overflow...

**Comment by minhnguyenwp** [comment]
Hi @samii-ullah  could you try this. is this the solution for your question: https://codesphere.com/articles/create-your-own
